win32ss/gdi/eng/pdevobj.h

```diff
@@ -201,4 +201,11 @@ PDEVOBJ_pdmMatchDevMode(
     PPDEVOBJ ppdev,
     PDEVMODEW pdm);
 
+FORCEINLINE
+VOID
+PDEVOBJ_vReference(PPDEVOBJ ppdev)
+{
+    InterlockedIncrement(&ppdev->cPdevRefs);
```

Comment:
Would it make sense to ASSERT(ppdev) here?

Reply:
I don't think that is needed. Think of these functions like C++ member functions. You wouldn't assert(this != NULL) there. And it would crash anyway with a NULL ppdev.